content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer & Uint.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.DT
+  - ib-19.3.Datatype-Primitief-Integer.DT
+  - ib-19.2.Datatype-Primitief-Uint.DT
+  - ib-19.3.Datatype-Primitief-Uint.DT
+---
+
+> Meer informatie over [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]]
+
+> Meer informatie over [[2. Uint|Uint]]
+
+## Opdracht 1
+Voeg het initialiseren toe aan het volgende codeblok. Maak gebruik van het integer datatype.
+
+### Specificaties:
+- Maak een variabele `age`.
+- Geef `age` de waarde `23`.
+- Print `age`.
+
+### Verwachte output:
+Als `age = 23, komt in de console:
```

Comment:
Opmerking: Is dit een verloren '  of is deze zijn partner kwijt?

Reply:
ze zijn herenigt:)